model/otlp/json_test.go

```diff
@@ -88,7 +88,7 @@ func TestMetricsJSON(t *testing.T) {
 }
 
 func TestLogsJSON(t *testing.T) {
-	encoder := newJSONMarshaler()
+	encoder := NewJSONLogsMarshaler()
```

Comment:
This doesnt appear related to the rest of the change.

Reply:
Done.